data-extraction/pyDataExtraction/__main__.py

```diff
@@ -0,0 +1,10 @@
+from json import dumps
+from typing import Union, Dict, Optional
+from abc import ABC, abstractmethod
+from pyDataExtraction.commonTypes.Graph import Graph
+
```

Comment:
add newline

Reply:
We can run flake8(for linting) and black(code formatting) in local(console) to do all such changes automatically and consistency will be maintained for all the contributors.

Reply:
I currently use black for formatting, though I've never used flake8, nor have I used black from the commandline. I know there's a precommit plugin for git that would call both to on attempted commits to make sure that they are used.

Reply:
You just have to install black in local(or in the virtual environment of python) and run `black <file_name>` This will do the formatting for the file.
And about flake8, It's just that we can have a consistent style guide for python over this project which follows pep-8 guidelines.
As some of the comments by @mvoitko  is regarding this only.
Just a suggestion 😄